graphics/jgraph/Portfile

```diff
@@ -3,13 +3,13 @@
 PortSystem          1.0
 
 name                jgraph
-version             8.3
+version             8.5
 categories          graphics
 # see http://packages.debian.org/changelogs/pool/main/j/jgraph/current/copyright
-license             GPL
+license             BSD-3-Clause
```

Comment:
for MacPorts the valid license description here is `BSD` which includes both the 2- and 3-clause